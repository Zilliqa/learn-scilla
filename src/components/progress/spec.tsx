import React from 'react';
import Progress from '.';
import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Progress value={5} max={10} color={'secondary'} />).toJSON();
  expect(tree).toMatchSnapshot();
});
