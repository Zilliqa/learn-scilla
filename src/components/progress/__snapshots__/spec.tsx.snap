// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="progress"
  style={
    Object {
      "height": 5,
    }
  }
>
  <div
    aria-valuemax={10}
    aria-valuemin={0}
    aria-valuenow={5}
    className="progress-bar bg-secondary"
    role="progressbar"
    style={
      Object {
        "width": "50%",
      }
    }
  />
</div>
`;
